src/main/kotlin/io/github/moulberry/notenoughupdates/commands/misc/ProfileViewerCommands.kt

```diff
@@ -37,6 +37,10 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 class ProfileViewerCommands {
     companion object {
         fun CommandContext<ICommandSender>.openPv(name: String) {
+            if (!NotEnoughUpdates.INSTANCE.isOnSkyblock) {
+                Minecraft.getMinecraft().thePlayer.sendChatMessage("/pv")
```

Comment:
```suggestion
                Minecraft.getMinecraft().thePlayer.sendChatMessage("/pv $name")
```

Reply:
Actually, this wouldnt work since then just /pv would be replaced with /pv <yourname>. But yeah, this breaks arguments for /pv

Reply:
i do not know how to fix this without getting bullied for doing goofy shit to brigadier